provision/minikube/measure/compute-stats.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+dataDir=${1:-"data"}
+
+for csvFile in $(find "${dataDir}" -path "**/keycloak-*/**" -name memory-usage-cgroups.csv -o -name memory-usage-rss.csv -o -name startup-times.csv ); do
```

Comment:
is there any dependency on other script to be run first for this to be useful ? I am not entirely clear of the sequence of scripts to be consumed in a right order